packages/iris-grid/src/mousehandlers/IrisGridCopyCellMouseHandler.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Grid,
+  GridMouseHandler,
+  GridPoint,
+  EventHandlerResult,
+  GridMouseEvent,
+} from '@deephaven/grid';
+import { ContextActionUtils } from '@deephaven/components';
+import IrisGrid from '../IrisGrid';
+
+class IrisGridCopyCellMouseHandler extends GridMouseHandler {
+  private irisGrid: IrisGrid;
+
+  constructor(irisGrid: IrisGrid) {
+    super();
+
+    this.irisGrid = irisGrid;
+    this.cursor = null;
+  }
+
+  onClick(
+    gridPoint: GridPoint,
+    grid: Grid,
+    event: GridMouseEvent
+  ): EventHandlerResult {
+    if (
+      event.altKey &&
+      !ContextActionUtils.isModifierKeyDown(event) &&
+      !event.shiftKey
+    ) {
+      this.cursor = null;
+      if (gridPoint.columnHeaderDepth !== undefined) {
+        this.irisGrid.copyColumnHeader(gridPoint.column);
+      } else {
+        this.irisGrid.copyCell(gridPoint.column, gridPoint.row);
+      }
+      return true;
+    }
+    return false;
+  }
+
+  onMove(
+    gridPoint: GridPoint,
+    _grid: Grid,
+    event: GridMouseEvent
+  ): EventHandlerResult {
+    if (
```

Comment:
We should be checking if the user `canCopy` here (on the `irisGrid.props`. @dsmmcken should we display a different cursor if copy is not available (ie. user doesn't have permissions)?

Reply:
Nah, no separate icon, just hide this feature.